Gopkg.toml

```diff
@@ -31,7 +31,7 @@
 
 [[constraint]]
   name = "github.com/minishift/minishift"
-  branch = "master"
+  version = ">=1.21.0"
```

Comment:
@agajdosi This is not a real fix IMO. Because when we move to minishift version 1.22.0 these tests will gain fail right?

Reply:
Or does it need to be same with the version of minishift binary is run for the tests.

Reply:
@LalatenduMohanty Probably `>=` will take care. Isn't it @agajdosi ?

Reply:
AFAIU yes, checked the `dep` docs once more and it seems like this will take each following release.

Reply:
@agajdosi why don't we use master branch itself so that nightly test for add-on also run as expected?